server/protocols/protocols-smtp/src/test/java/org/apache/james/smtpserver/SMTPServerTest.java

```diff
@@ -1420,7 +1420,7 @@ public void testAuthShouldFailedWhenUserPassIsNotBase64Decoded() throws Exceptio
         smtpProtocol.sendCommand("AUTH PLAIN");
         smtpProtocol.sendCommand("canNotDecode");
         assertThat(smtpProtocol.getReplyString())
-            .contains("501 Could not decode parameters for AUTH PLAIN");
+            .contains("535 Authentication Failed");
```

Comment:
Is this a regression?

Reply:
The method `decodeBase64(line)` can decode with any string. 
Or we will modify the old method

Reply:
I tried, but I can not determine the decodevalue is correct format or not.
Eg:
`new String(Base64.getDecoder().decode("Tung"))`  = `N��`

`Username` is not precondition check for this character, and it passed
__________

The reason is `regression` for this test case is before that we do not accept tokens when it'size =1